src/main/scala/scala/tools/refactoring/common/TreeTraverser.scala

```diff
@@ -412,5 +421,47 @@ trait TreeTraverser extends TracingImpl {
           super.traverse(t)
       }
     }
+
+    /*
+     * Refined types need special handling, since their original representation might
+     * lack symbols. This method takes care of this.
+     */
+    private def handleRefinedType(orig: Tree, tpe: RefinedType): Unit = {
+      orig.tryMatch {
+        case orig: RefTree if orig.symbol == NoSymbol =>
+          tpe.parents.foreach(handleParentTypeInRefinedType(_, orig))
+
```

Comment:
@wpopielarski see https://scala-ide-portfolio.assembla.com/spaces/scala-ide/tickets/1002857; I'll look into this in the not too distant future.